specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/definitions.json

```diff
@@ -1957,6 +1957,38 @@
         "circuitBreaker": {
           "$ref": "#/definitions/BackendCircuitBreaker",
           "description": "Backend Circuit Breaker Configuration"
+        },
+        "pool": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/BackendPool"
+            },
```

Comment:
this looks like its using allOf at the wrong level perhaps? Usually you use it inside a definition, not inside a property.